src/main/java/com/coherent/training/selenium/kapitsa/web/base/BrowserDriverFactory.java

```diff
@@ -0,0 +1,38 @@
+package com.coherent.training.selenium.kapitsa.web.base;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class BrowserDriverFactory {
+    private final ThreadLocal<WebDriver> driver = new ThreadLocal<>();
+    private final String browser;
+
+    public BrowserDriverFactory(String browser) {
+        this.browser = browser.toLowerCase();
+    }
+
+    public WebDriver createDriver(){
+        switch (browser){
+            case "chrome":
```

Comment:
drivers could get out of date , no need to keep them in the repo
also remove absolute paths